woocommerce/woocommerce
# Change WooCommerce Payments to WooPayments across the WC Core

### Changes proposed in this Pull Request:

Closes https://github.com/Automattic/woocommerce-payments-server/issues/3525.

This PR covers WC Core part of the branding rollout (WooCommerce Payments -> WooPayments). See the project thread here paJDYF-7Sw-p2.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Build a `woocommerce` plugin from this branch using `pnpm --filter=woocommerce run build:zip`.
2. Spin up a new JN site
3. Install WooCommerce from `.zip` which is built from step 1. 
4. You'll see profile wizard, skip for now (It'll be done in https://github.com/Automattic/woocommerce-payments-server/issues/3641). Select US as country and proceed.
5. On the home page, click Set Up payments. You should see updated brand name on banner and in the text
    ![image](https://github.com/woocommerce/woocommerce/assets/79862886/744430e7-d140-40f8-872e-9fcf11180023)
6. Visit WooCommerce -> Settings -> Payments. Click Install near the WooPayments plugin. p.s. plugin text is updated in https://github.com/Automattic/woocommerce-payments/pull/6581, and will be released on July, 19.
7. You'll be redirected to `payments/connect` page, please go back to WooCommerce -> Settings -> Payments and check that banner is also updated with new brand name (WooPayments)
    ![image](https://github.com/woocommerce/woocommerce/assets/79862886/4ab487e3-4737-4673-b155-cdf3c1efef3f)
8. Go to the WooCommerce -> Home and check that task looks good when the WooPayments plugin is installed 
![image](https://github.com/woocommerce/woocommerce/assets/79862886/69d243ab-f854-4509-8b33-3d8cc4640da7)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
